recipes/libonnx/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libonnx" %}
+{% set version = "1.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/onnx/onnx/archive/v{{ version }}.tar.gz
+  sha256: d68a5471bcd70eb23ace39717d0177a6273ab9ea51064892a0ba589b73b619ac
+
+build:
+  number: 0
+  skip:
+    # Needs C++11
+    - true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - protobuf
+    - libprotobuf
+    - ninja
+    - pybind11
+  run:
+    - python
+    - libprotobuf
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX {{ name }}  # [unix]
+    - conda inspect objects -p $PREFIX {{ name }}   # [osx]
```

Comment:
These are default now in conda-build. Can you add a real test? Or a check if the expected files are created?